### DISCUSSION TITLE
Open a new tab to external website in navigation

### DISCUSSION THREAD

**Discussion by psinger** [h2oai/wave#1204] [Ideas] [answered]
I would like to open a new tab opening an external website in my navigation using `nav_card` and `nav_item`.

I am only aware of Javascript workaround to make this work, but some users might not have pop-up function activated.

Would it be possible to add a new option for that? Or is there any other solution for me to do it right now?

**Comment by marek-mihok** [comment, marked as answer]
Sure, it's possible. We are about to add a new option for that. Tracked in #1952.
